BUILDS.md

```diff
@@ -132,6 +132,8 @@ Note: `minimal` variant is not listed as it shouldn't be used outside of the [up
 | USE_MPR121                | - | - / - | - | - | - | - |
 | USE_CCS811                | - | - / - | - | x | - | - |
 | USE_CCS811_V2             | - | - / x | - | - | - | - |
+| USE_ENS16x                | - | - / x | - | x | - | - |
+| USE_ENS210                | - | - / x | - | x | - | - |
```

Comment:
Do not add your drivers by default

Reply:
Disabled by default, memory information added